### PR TITLE
refactor: transfer to kiegroup organization

### PR DESCRIPTION
**Pull request by lampajr**
**Thank you for submitting this pull request**

fix _(please add the issue ID if it exists)_

### Checklist
- [ ] Tests added if applicable.
- [x] Documentation updated if applicable.

> **Note:** `dist/cli/index.js` and `dist/gha/index.js` are automatically generated by git hooks and gh workflows.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@lampajr/bper",
-  "version": "3.1.1",
+  "name": "@kie/bper",
```
Comment (Ginxo) [inline package.json:2]: wouldn't you call it just simply git-backporting? :thinking: 
```suggestion
  "name": "@kie/git-backporting",
```

**Review by Ginxo — APPROVED** [review]
LGTM, just a suggestion

File: package.json (hunk 1 above)
Comment (lampajr) [reply package.json:2]: oh you are right, I missed to rename that - going to update README as well

Comment (lampajr) [reply package.json]: updated!

**Pull request merged**